pkg/genericcomp/genericcomp.go

```diff
@@ -170,3 +182,21 @@ func IsFluentdEnabled(generic *v1beta1v8o.VerrazzanoGenericComponent) bool {
 
 	return *generic.FluentdEnabled
 }
+
+// Get the port and targetPort values
+func getPorts(generic *v1beta1v8o.VerrazzanoGenericComponent) (int32, int32) {
```

Comment:
since callers of this function only use the targetPort there is no need to return the HostPort and the name of the function should be getTargetPort.

Reply:
Hey @rdonat  fixed